### PR TITLE
Selection controls

### PR DESCRIPTION
**Pull request by bdlukaa**
## Pre-launch Checklist

- [x] I have run `dartfmt` on all changed files  
- [x] I have updated `CHANGELOG.md` with my changes  
- [x] I have run "optimize/organize imports" on all changed files
- [x] I have addressed all analyzer warnings as best I could
- [x] I have added/updated relevant documentation
- [ ] I have run `flutter pub publish --dry-run` and addressed any warnings

### CODE REVIEW

File: lib/src/controls/form/selection_controls.dart [hunk 1]
```diff
@@ -119,9 +117,7 @@ class _FluentTextSelectionControlsToolbar extends StatefulWidget {
 }
 
 class _FluentTextSelectionControlsToolbarState
-    extends State<_FluentTextSelectionControlsToolbar>
-    with SingleTickerProviderStateMixin {
-  late AnimationController _ac;
```
Comment (bdlukaa) [inline lib/src/controls/form/selection_controls.dart:124]: Removed the animation controller since it's no longer used on Win UI 3

**Review by WinXaito — APPROVED** [review]
Great work !

The cursor color doesn't follow accent color: 
cursor_color

And: cursorHeight property has no effect

And the others point are certainly flutter-related:

- Customize cursorBlink speed ?
- Triple clic selection
- Right clic outside the selection deselect the content


------------------

And maybe doc ? 😅

**Comment by bdlukaa** [comment]
> The cursor color doesn't follow accent color:

It should be the inactive color tho

**Comment by WinXaito** [comment]
> It should be the inactive color tho

Anyway, just no "blue" if the accent color is other ;)

**Pull request merged**